src/display/editor/freetext.js

```diff
@@ -39,8 +42,16 @@ class FreeTextEditor extends AnnotationEditor {
     this.#fontSize = params.fontSize || 10;
   }
 
-  static setL10n(l10n) {
+  static initialize(l10n) {
     this._l10nPromise = l10n.get("freetext_default_content");
+    document.documentElement.style.setProperty(
+      "--freetext-line-height",
+      LINE_FACTOR
+    );
+    document.documentElement.style.setProperty(
+      "--freetext-padding",
+      `${INTERNAL_PADDING}px`
+    );
```

Comment:
How about simply *reading* the padding value as defined in the CSS file, rather than overwriting it here (with a value it should already have)?

Something like this could perhaps work:
```javascript

const style = getComputedStyle(document.documentElement);
this.internalPadding = parseInt(
  style.getPropertyValue("--freetext-padding"),
  10
);
```